src/FileManager.java

```diff
@@ -0,0 +1,110 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+public class FileManager {
+
+    private static Path currentDirectory = Paths.get(System.getProperty("user.dir"));
+
+    public static void run() {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        String command;
+        
+        while (true) {
+            System.out.println(currentDirectory.toAbsolutePath() + " $ ");
+            try {
+                command = reader.readLine();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+
+            if (command.equals("exit")) {
+                break;
+            }
+
+            String[] keys = command.split("\\s+");
+
+            switch (keys[0]) {
+                case "cd":
+                    changeDirectory(keys);
+                    break;
+                case "cp":
+                    copyFile(keys);
+                    break;
+                case "ls":
+                    localStorage(keys);
+                    break;
+                case "pwd":
+                    printWorkingDirectory(keys);
+                    break;
+                default:
+                    System.out.println("The wrong command is entered");
+                    break;
+            }
+        }
+    }
+
+    private static void changeDirectory(String[] keys) {
+        if (keys.length == 1) {
+            System.out.println("Specify the directory");
+        }
+
+        String targetDirectory = keys[1];
+
+        if (targetDirectory.equals("..")) {
+            Path parentDirectory = currentDirectory.getParent();
+            if (parentDirectory == null) {
+                System.out.println("You are already in the root directory");
+            }
+            currentDirectory = parentDirectory;
+        } else if (targetDirectory.startsWith("/")) {
+            currentDirectory = Paths.get(targetDirectory);
+        } else {
+            currentDirectory = currentDirectory.resolve(targetDirectory);
+        }
+
+        if (!Files.exists(currentDirectory)) {
```

Comment:
I believe you should check first if the directory exists, and then, if it exists, continue running your logic, otherwise return from the method.